pkg/scheduler/scheduler.go

```diff
@@ -45,12 +45,15 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/metrics"
 	"k8s.io/kubernetes/pkg/scheduler/util"
 
+	"errors"
```

Comment:
I think move this line to the beginning of the `import` clause is more consistent with Golang's  idiomatic.